--- trusty/vendor/google/aosp/scripts/build.py.oirg	2020-07-03 13:37:52.964890368 -0400
+++ trusty/vendor/google/aosp/scripts/build.py	2020-07-03 13:44:03.863315217 -0400
@@ -119,7 +119,7 @@
         cmd += "; export BUILDID=" + args.buildid
         if args.clang is not None:
             cmd += "; export CLANGBUILD=" + str(args.clang).lower()
-        cmd += "; nice make " + project + " -j " + str(args.jobs)
+        cmd += "; nice bear make " + project + " -j " + str(args.jobs)
         # Call envsetup.  If it fails, abort.
         cmd = "source %s && (%s)" % (os.path.join(script_dir, "envsetup.sh"),
                                      cmd)
